packages/odie-client/src/components/message/style.scss

```diff
@@ -222,7 +222,12 @@ $blueberry-color: #3858e9;
 			&:not( .is-expanded ) {
 				.foldable-card__content {
 					border: none;
-					margin-top: 0;
+				}
+			}
+
+			&.is-expanded {
+				.foldable-card__content {
+					max-height: 350px;
```

Comment:
I duplicated some of the source elements and made it still cropped.
<img width="1027" height="511" alt="Screenshot 2025-10-27 at 11 57 04" src="https://github.com/user-attachments/assets/1c687a9a-0bc6-4343-9b3e-45688a41a792" />

Maybe we could do:
```suggestion
					max-height: fit-content;
```